var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+from spack import *
+from spack.pkg.builtin.neurodamus_model import NeurodamusModel
+
+
+class NeurodamusHippocampus(NeurodamusModel):
+    """Neurodamus with built-in Hippocampus model.
+    """
+
+    homepage = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
+    git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
+
+    version('develop', git=git, branch='master', submodules=True)
+    version('1.0', tag='1.0', submodules=True)
```

Comment:
1.0 -> 0.1 ?

Reply:
After years of devel, these models don't deserve 1.0?

Reply:
0.1 feels cortex mods are in very early stage. If you are not confident with 1.0 we can go `1.0-alpha`, 2019.01, 0.9.... You name it :)

Reply:
It's a little detail. I also agree that 1.0 might be too much, sounds very stable. I don't dislike by `yyyy.mm` scheme though. Maybe that's even the best option?